content/en/synthetics/private_locations/configuration.md

```diff
@@ -13,165 +13,24 @@ further_reading:
-: **Type**: Number <br>
-**Default**: `10`<br>
-Maximum number of tests executed in parallel.
-
-`maxTimeout`
-: **Type**: Number <br>
-**Default**: `60000`<br>
-Maximum test execution duration for API tests (in milliseconds).
-
-`enableStatusProbes`
-: **Type**: Boolean <br>
-Enables the readiness and liveness of private location probes. This enables two endpoints: http://127.0.0.1:8080/liveness and http://127.0.0.1:8080/readiness.
-
-`statusProbesPort`
-: **Type**: Number <br>
-**Default**: `8080`<br>
-Overrides the port for the private location status probes.
-
-`maxNbRedirects`
-: **Type**: Number <br>
-**Default**: `20`<br>
-Maximum number of redirections for HTTP and WebSocket tests.
-
-`variableOverride`
-: **Type**: String <br>
-Overrides the variables used in tests running on the Private Location. Format: `VARIABLE=value`.
-All variables imported this way are obfuscated.
-
-`environmentVariableOverride`
-: **Type** String <br>
-Overrides variables used in tests running on the Private Location with environment variables. It requires the environment variables to be imported in the containerized environment.
-Example for Docker: `docker run --env VARIABLE gcr.io/datadoghq/synthetics-private-location-worker --environmentVariableOverride VARIABLE`.
-All variables imported this way are obfuscated. 
-
-## Private root certificates
-
-You can upload custom root certificates to your private locations to have your API and Browser tests perform SSL handshake using your own `.pem` files. When spinning up your private location containers, mount the relevant certificate `.pem` files to `/etc/datadog/certs`, the same way your private location configuration file is mounted. These certificates are then considered trusted CA and used as such at test runtime.
-
-**Note**: This feature is supported for versions 1.5.3+ of the private location Docker image.
-
-## Private locations admin
-
-`config`
-: **Type**: String <br>
-**Default**: `/etc/datadog/synthetics-check-runner.json`<br>
-Path to JSON configuration file.
-
-`logFormat`
-: **Type**: String <br>
-**Default**: `pretty`<br>
-Format log output between `"pretty"`, and `"json"`. Setting your log format to `json` allows you to have these logs automatically parsed when collected by Datadog.
-
-`verbosity`
-: **Type**: Number <br>
-**Default**: `3`<br>
-Verbosity level (for example, `-v`, `-vv`, `-vvv`, ...).
-
-`dumpConfig`
-: **Type**: Boolean <br>
-**Default**: `none`<br>
-Display worker configuration parameters without secrets.
-
-`dumpFullConfig`
-: **Type**: Boolean <br>
-**Default**: `none`<br>
-Display full worker configuration parameters.
-
-`help`
-: **Type**: Boolean <br>
-**Default**: `none`<br>
-Show help.
-
-**Note**: Private Location containers output logs to stdout/stderr without saving them within the container.
+Arguments set in the launch command have precedence over the configuration file. However, these options are not stored and are consequently only prevalent for a given launch.
```

Comment:
I know you didn't change the above text in this PR, but should "prevalent" be "relevant"?

Reply:
```suggestion
Arguments set in the launch command have precedence over the configuration file. However, these options are not stored and are consequently only revelant for a given launch.
```